packages/asini-changelog/.eslintrc

```diff
@@ -1,40 +1,3 @@
 {
-  "parser": "babel-eslint",
-  "extends": "eslint:recommended",
-  "rules": {
-    "quotes": [
-      2,
-      "double",
-      "avoid-escape"
-    ],
-    "no-var": 0,
-    "keyword-spacing": 2,
-    "strict": 0,
-    "no-underscore-dangle": 0,
-    "curly": 0,
-    "no-multi-spaces": 0,
-    "key-spacing": 0,
-    "no-return-assign": 0,
-    "consistent-return": 0,
-    "no-shadow": 0,
-    "comma-dangle": 0,
-    "no-use-before-define": 0,
-    "no-empty": 0,
-    "new-parens": 0,
-    "no-cond-assign": 0,
-    "no-fallthrough": 0,
-    "new-cap": 0,
-    "no-loop-func": 0,
-    "no-unreachable": 0,
-    "no-labels": 0,
-    "no-process-exit": 0,
-    "camelcase": 0,
-    "no-console": 0,
-    "no-constant-condition": 0,
-    "no-inner-declarations": 0,
-    "indent": ["error", 2]
-  },
-  "env": {
-    "node": true
-  }
+  "extends": "asini"
```

Comment:
Yeah, this is just to adhere to the ideal of keeping packages in an Asini repo independently configured.  To the extent possible I'd like to be able to take a package from the repo, move it out, and have it still work.  What I want _Asini_ to do for me is then use that independent configuration in a smart way to avoid unnecessary duplicate effort.  So in this case although we're referencing our shared config in each package we don't need to re-download `eslint-config-asini` a bunch of times because our packages are in an Asini repo.